panoptic_mapping/src/labels/range_label_handler.cpp

```diff
@@ -0,0 +1,47 @@
+#include "panoptic_mapping/labels/range_label_handler.h"
+
+#include <iomanip>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "panoptic_mapping/tools/coloring.h"
+
+namespace panoptic_mapping {
+
+config_utilities::Factory::RegistrationRos<LabelHandlerBase, RangeLabelHandler>
+    RangeLabelHandler::registration_("range");
+
+void RangeLabelHandler::Config::setupParamsAndPrinting() {
+  setupParam("verbosity", &verbosity);
+  setupParam("num_labels", &num_labels);
+}
+
+void RangeLabelHandler::Config::checkParams() const {
+  checkParamCond(num_labels >= 0, "num_labels must not be negative.");
```

Comment:
This could also be a  `checkParamGT(num_labels, 0, "num_labels");` does not change much but more compliant with the standard use :)